src/pages/landing/feed/components/RewardActivityItem.tsx

```diff
@@ -63,55 +63,40 @@ const RewardItem = ({
   return (
     <Box
       backgroundColor="neutral.50"
-      border="2px"
-      borderColor="neutral.200"
+      border='2px'
+      borderColor='neutral.200'
       borderRadius={12}
       mt={2}
       p={3}
       pos={'relative'}
       width={'100%'}
     >
-      <Stack direction="row">
-        <Box borderRadius={12} overflow={'hidden'} width="70px">
-          <div
-            style={{
-              display: 'block',
-              position: 'relative',
-              paddingTop: '100%',
-              width: '100%',
-            }}
-          >
-            <div
-              style={{
-                display: 'block',
-                position: 'absolute',
-                top: 0,
-                left: 0,
-                right: 0,
-                bottom: 0,
-                background: `transparent url(${reward.image}) no-repeat center center / cover`,
-              }}
-            ></div>
-          </div>
-        </Box>
-        <Stack direction="column" flex={1} pl={2} gap={0.25}>
-          <Text fontWeight={700} fontSize={16} color="neutral.900">
-            {reward.rewardName}
-          </Text>
-          <Text fontSize={12} color="neutral.600">{`${
-            reward.maxClaimable && reward.maxClaimable > 0
-              ? reward.maxClaimable - reward.sold + ` ${t('remaining')}, `
-              : ''
-          }${reward.sold} ${t('sold')}`}</Text>
+        <Stack direction="row">
+            <Box borderRadius={12} overflow={'hidden'} width="70px">
+                <div style={{display: 'block', position: 'relative', paddingTop: '100%', width: '100%'}}>
+                    <div style={{display: 'block', position: 'absolute', top: 0, left: 0, right: 0, bottom: 0, background: `transparent url(${reward.image}) no-repeat center center / cover`}}>
```

Comment:
Because I am adding a lot of custom styling to force the behavior of the background image being a perfect square, so I figured div would be fine.  Do we have something against using divs?  Chakra renders down to divs anyway, but then you have to battle any default styling.

Reply:
Yeah, it's okay for normal stuff but then again when you need themed settings, it becomes tedious.
Chakra provides Box as a basic no style div, you can use that .